scripts/gitlab_trigger_client_publish

```diff
@@ -5,6 +5,9 @@ set -e
 # Number of release branches to build when trigger comes from meta-mender
 NUMBER_OF_MINOR_VERSIONS=3
 
+# Branch to use when building QEMU images to publish
+LATEST_STABLE_YOCTO_BRANCH="dunfell"
```

Comment:
This is fine, but to score extra points, and never have to look at this again, I'd consider setting this to the same value as `.gitlab-ci.yml: POKY_REV`.

Reply:
Thanks. Please see rework.